code/modules/client/preference_setup/loadout/loadout_shoes.dm

```diff
@@ -65,42 +70,6 @@
 	display_name = "shoes, red"
 	path = /obj/item/clothing/shoes/red
 
-/datum/gear/shoes/white
-	display_name = "shoes, white"
-	path = /obj/item/clothing/shoes/white
-
-/datum/gear/shoes/yellow
```

Comment:
Undocumented change not mentioned in the PR. Outright removed since normal colorable shoe are not a thing.